plugins/feeds/public/CertPaIT.py

```diff
@@ -25,15 +27,23 @@ class CertPaIT(Feed):
     )
 
     re_generic_details = re.compile('<p>Filename: <b>(?P<filename>.*)\\</b\\><br>Filetype: (?P<filetype>.*)</p>')
-    
+
     def update(self):
+
+        since_last_run = datetime.now(timezone('UTC')) - self.frequency
+
         for item in self.update_xml('item', ['title', 'link', 'pubDate', 'description']):
-            self.analyze(item)
+            pubDate = parse_date_to_utc(item['pubDate'])
```

Comment:
Please stick to the snake_case format for variables in your code (`pub_date` or `pubdate`)